KeyboardAvoidanceSwiftUI/KeyboardHeightPublisher.swift

```diff
@@ -9,16 +9,22 @@
 import Combine
 import UIKit
 
+fileprivate var keyboardUpdates: AnyCancellable?
+fileprivate let keyboardHeightPublisher: CurrentValueSubject<CGFloat, Never> = {
```

Comment:
We change how we're publishing the keyboard height so that `KeyboardAdaptive` can read the current value when re-rendering _not_ in response to the keyboard changing, but rather in response to another condition.

I tried caching the keyboard height on the `KeyboardAdaptive` modifier e.g. in a `@State` property and it didn't work but rather got reset, I would assume when the view hierarchy re-renders.